grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java

```diff
@@ -128,48 +118,44 @@ public static GraqlQueryException insertRelationWithoutType() {
         return new GraqlQueryException(ErrorMessage.INSERT_RELATION_WITHOUT_ISA.getMessage());
     }
 
-    public static GraqlQueryException insertUnsupportedProperty(String name, Schema.MetaSchema metaSchema) {
-        return new GraqlQueryException(INSERT_UNSUPPORTED_PROPERTY.getMessage(name, metaSchema.getLabel()));
-    }
-
     public static GraqlQueryException insertPredicate() {
         return new GraqlQueryException(ErrorMessage.INSERT_PREDICATE.getMessage());
     }
 
-    public static GraqlQueryException insertIsaAndSub(String printableName) {
-        return new GraqlQueryException(INSERT_ISA_AND_SUB.getMessage(printableName));
-    }
-
     public static GraqlQueryException insertRecursive(VarPatternAdmin var) {
         return new GraqlQueryException(INSERT_RECURSIVE.getMessage(var.getPrintableName()));
     }
 
-    public static GraqlQueryException insertInstanceWithLabel(Label label) {
-        return new GraqlQueryException(INSERT_INSTANCE_WITH_NAME.getMessage(label));
-    }
-
-    public static GraqlQueryException insertWithoutType(ConceptId conceptId) {
-        return new GraqlQueryException(INSERT_WITHOUT_TYPE.getMessage(conceptId));
-    }
-
     public static GraqlQueryException insertUndefinedVariable(VarPatternAdmin var) {
         return new GraqlQueryException(INSERT_UNDEFINED_VARIABLE.getMessage(var.getPrintableName()));
     }
 
-    public static GraqlQueryException createInstanceOfMetaConcept(VarPatternAdmin var, Type type) {
+    public static GraqlQueryException createInstanceOfMetaConcept(Var var, Type type) {
         return new GraqlQueryException(var + " cannot be an instance of meta-type " + type.getLabel());
     }
 
     public static GraqlQueryException insertMetaType(Label label, OntologyConcept ontologyConcept) {
         return new GraqlQueryException(ErrorMessage.INSERT_METATYPE.getMessage(label, ontologyConcept.getLabel()));
     }
 
-    public static GraqlQueryException insertMultipleValues(ValuePredicateAdmin predicate, Object value) {
-        return new GraqlQueryException(INSERT_MULTIPLE_VALUES.getMessage(value, predicate));
+    public static GraqlQueryException insertMultipleProperties(String property, Object value1, Object value2) {
```

Comment:
Take this more as a suggestion than a "must change" but for each exception I like to javadoc what conditions result in the exception being thrown. In many cases the title of the method or the contents give away the condition but I still think it's nice to have it in clear text. For example:

```
    /**
     * Thrown when a {@link Thing} is not allowed to have {@link Resource} of that {@link ResourceType}
     */
    public static GraphOperationException hasNotAllowed(Thing thing, Resource resource)
```